graphql_compiler/schema_transformation/utils.py

```diff
@@ -148,7 +152,7 @@ def get_scalar_names(schema):
     scalars = {
```

Comment:
Consider renaming this variable name, `scalars` is only telling part of the story now.